packages/worker/src/tests/structures/groups.ts

```diff
@@ -18,7 +18,7 @@ export const UserGroup = () => {
     apps: [],
     color: generator.color(),
     icon: generator.word(),
-    name: generator.word({ length: 2 }),
+    name: generator.word(),
```

Comment:
This was causing flakiness, hopefully it will remove all the need rerun issues

Reply:
nice!